mail-filter/sqlgrey/files/sqlgrey.service

```diff
@@ -3,7 +3,19 @@ Description=SQLgrey Postfix Grey-listing Policy service
 After=network.target
 
 [Service]
+User=sqlgrey
+Group=sqlgrey
 ExecStart=/usr/sbin/sqlgrey
+PermissionsStartOnly=true
+ExecStartPre=/usr/bin/touch /run/sqlgrey.pid && /bin/chown sqlgrey:sqlgrey /run/sqlgrey.pid
```

Comment:
I rethought this and I don't think `PIDFile` needs to be specified, so /run/sqlgrey.pid also need not be created. I've updated the PR accordingly.